.pre-commit-config.yaml

```
@@ -0,0 +1,31 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v5.0.0
+    hooks:
+      - id: trailing-whitespace
+        exclude: ^testing/Baseline
+      - id: end-of-file-fixer
+        exclude: ^testing/Baseline
+
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.5.5
```

Let's use the current version.

```suggestion
    rev: v0.8.1
```